src/senaite/core/upgrade/v02_06_000.py

```diff
@@ -265,3 +266,31 @@ def update_workflow_role_mappings(wf_id, objs_or_brains):
         workflow.updateRoleMappingsFor(obj)
         obj.reindexObject()
         obj._p_deactivate()
+
+
+def remove_folders_snapshots(tool):
+    """Removes the auditlog snapshots for portal and setup folders and remove
+    the IAuditable marker interface as well
+    """
+    logger.info("Removing snapshots from portal and setup folders ...")
+    portal = tool.aq_inner.aq_parent
+    bika_setup = api.get_setup()
+    setup = api.get_senaite_setup()
+
+    # pick all folders "folders"
+    folders = portal.objectValues()
+    folders += bika_setup.objectValues()
+    folders += setup.objectValues()
+
+    # remove non-objects
+    folders = filter(api.is_object, folders)
+    folders = list(set(folders))
+
+    # remove setup folders (they hold settings as fields)
+    skip = [bika_setup, setup]
+    folders = filter(lambda folder: folder not in skip, folders)
+
+    # disable auditlog and snapshots
+    map(disable_snapshots, folders)
```

Comment:
Better use an explicit API function instead using the the one from `setuphandlers`

Reply:
Done with https://github.com/senaite/senaite.core/pull/2479/commits/94ff2a92c1be84c8c433bad3e53216c150044de4